ui/src/components/dialog/QasDialog.vue

```diff
@@ -148,7 +152,25 @@ export default {
 
   methods: {
     async submitHandler () {
-      this.useForm && this.$emit('validate', await this.$refs.form.validate())
+      if (!this.useForm) return
+
+      if (this.useValidationAllAtOnce) {
+        let isAllComponentValid = true
+        const components = this.$refs.form.getValidationComponents() || []
+
+        for (const component of components) {
+          const isValid = component?.validate?.()
```

Comment:
É possível o `component` vir `undefined`? Pois acho que não precisaria do: `component?` aqui...

Reply:
Será que daria para usar um `.every()` invés do `for of`?

Reply:
Por isto existe o optional chaining. E sobre o every não, já testei.